File: Basic of Python/prime number up to n.py
a=int(input("Enter the number:"))
flag=0
for i in range(2,a+1):
    b=i//2+1
    for j in range(2,b):
        if i%j==0:
            flag=1
    if flag==0:
        print("This number is a prime number:",i)
    flag=0

File: c programming/python assignment 10 qus but in c/prime_number_up_to_n.c
#include <stdio.h>

int main(){
    int n,flag=0;
    printf("Enter the number:");
    scanf("%d",&n);
    for(int i=2;i<=n;i++){
        for (int j=2;j<=i/2;j++){
            if (i%j==0){
                flag=1;
            }
        }
        if (flag==0){
            printf("This %d is a prime number\n",i);
        }
    flag=0;
    }
    return 0;
}